src/preprocessor-unpickle-reference.py

```diff
@@ -0,0 +1,45 @@
+import sys
+import pandas as pd
+import numpy as np
+import click
+import matplotlib.pyplot as plt
+import altair as alt
+import dataframe_image as dfi
+from sklearn.preprocessing import (
+    OneHotEncoder,
+    OrdinalEncoder,
+    StandardScaler,
+    FunctionTransformer,
+    
+)
+import pickle
+from sklearn.impute import SimpleImputer
+from sklearn.compose import make_column_transformer
+from sklearn.pipeline import Pipeline, make_pipeline
+
+from correlation_matrix import pearson_correlation_matrix
+from plot_histogram_with_exclusions import plot_histogram_with_exclusions
+
+@click.command()
+@click.option('--training-data', type=str, help="Path to training data ex: data/raw/")
+@click.option('--pickle-file', type=str, help="Path to where preprocessor should be written ex: results/models/preprocessor" )
+
+def map_to_other(df, categories_list):
```

Comment:
You do not need categories_list anymore as it is the same as `education_order`, and this is hardcoded.